content/source/docs/enterprise/api/teams.html.md

```diff
@@ -34,31 +34,31 @@ $ curl \
 {
   "data": [
     {
-      "id": "257529",
+      "id": "team-6p5jTwJQXwqZBncC",
```

Comment:
Oof, the benefits of replacing the whole response object from time to time. 😬 good catch!